packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -45,8 +45,10 @@ contract DeployConfig is Script {
     uint256 public l2GenesisBlockBaseFeePerGas;
     uint256 public gasPriceOracleOverhead;
     uint256 public gasPriceOracleScalar;
+    bool public enableGovernance;
     uint256 public eip1559Denominator;
     uint256 public eip1559Elasticity;
+    uint256 public l1GenesisBlockTimestamp;
```

Comment:
Depending on the resolution of [this comment](https://github.com/ethereum-optimism/optimism/pull/9292/files#r1473559740), this may not be needed